src/core/server/base.js

```diff
@@ -433,7 +433,7 @@ function baseServer(
 
       // We need to render once because it will force components to
       // dispatch data loading actions which get processed by sagas.
-      _log.info('First component render to dispatch loading actions');
+      _log.debug('First component render to dispatch loading actions');
```

Comment:
was there any logic behind which log statements you downgraded?  (I do see `First component render....` and `Second component render....` a lot in the logs so these are good candidates for sure)

Reply:
In most cases, we need the log statements during development but all the ones I changed are not helpful in production. Most of them (all but the ones at the end of this file and in `src/amo/components/HeroRecommendation/index.js`) are sent to Stackdriver on **every** page load.